app/controllers/api/v0/articles_controller.rb

```diff
@@ -16,7 +16,7 @@ class ArticlesController < ApiController
         title description main_image published_at crossposted_at social_image
         cached_tag_list slug path canonical_url comments_count
         public_reactions_count created_at edited_at last_comment_at published
-        updated_at video_thumbnail_url
+        updated_at video_thumbnail_url reading_time
```

Comment:
it might make me medium happy if this were alphabetized but I probably worry about that too much - suffice it to say I did wonder _where_ to add the new attribute to the list for more than a second.